refinery/data_set_manager/utils.py

```diff
@@ -615,12 +613,8 @@ def _generate_solr_params(params, assay_uuids=[]):
                   'facet.limit=-1'
                   ])
 
-    solr_params = 'fq=({})'.format(
-        ' OR '.join(map(
-            lambda id: 'assay_uuid:{}'.format(id),
-            assay_uuids
-        ))
-    )
+    assert len(assay_uuids) > 0, 'At least one assay must be accessible'
```

Comment:
Just committed a fix I like: We're inside a method which is building the query string. rather than using errors, we just return None is there query string would be invalid, and then at the top level that can be handled.